external/loaders/loaders/mappers/_nudged/_nudged.py

```diff
@@ -124,6 +126,7 @@ def open_nudge_to_fine(
         "nudging_tendencies.zarr",
         "state_after_timestep.zarr",
     ),
+    cache_size_mb: float = None,
```

Comment:
Should be Optional if default is None (how does this pass mypy checks?), can also be integer since this gets cast to an integer anyways when it's used, and I doubt we'd want to specify fractional MB.
```suggestion
    cache_size_mb: Optional[int] = None,
```